client/src/features/action/assets/Assets.tsx

```diff
@@ -133,6 +151,10 @@ function Assets(): JSX.Element {
           </Paper>
           <NeedsAndOffers headerText="Nonprofit Needs" assets={mockData} />
           <NeedsAndOffers headerText="Offers" assets={mockData} />
+          <NeedsAndOffers
+            headerText="Test"
+            assets={selectedAssetType === 'donation' ? donations : needs}
+          />
```

Comment:
let's just remove the Offers section altogether and make yours have `headerText="Offers"`